src/components/ContextualMenu/ContextualMenu.scss

```diff
@@ -87,4 +87,27 @@ $ContextualMenu-iconWidth: 24px;
   @include right(8px);
   width: 16px;
   text-align: center;
+}
+
+// The following is temporary to allow the React Contextual Menu to appear correctly.
```

Comment:
Is this comment still relevant? We should avoid including references to internal projects/files/issues where possible.